Foundation/NSJSONSerialization.swift

```diff
@@ -97,7 +97,26 @@ public class NSJSONSerialization : NSObject {
     /* Generate JSON data from a Foundation object. If the object will not produce valid JSON then an exception will be thrown. Setting the NSJSONWritingPrettyPrinted option will generate JSON with whitespace designed to make the output more readable. If that option is not set, the most compact possible JSON will be generated. If an error occurs, the error parameter will be set and the return value will be nil. The resulting data is a encoded in UTF-8.
      */
     public class func dataWithJSONObject(obj: AnyObject, options opt: NSJSONWritingOptions) throws -> NSData {
-        NSUnimplemented()
+        guard obj is NSArray || obj is NSDictionary else {
```

Comment:
Isn't this going to fail for e.g. `[String]` or `[String:Any]`? Because those are not automatically bridged to `NSArray` and `NSDictionary` on Linux, so unless things have changed, I don't think this will work on Linux.